src/main/java/de/unijena/cheminf/mortar/gui/views/SettingsView.java

```diff
@@ -206,15 +207,22 @@ private void addPropertyItems(GridPane aGridPane, List<Property<?>> aPropertiesL
             aRecentPropertiesMap.put(tmpPropName, tmpRecentValue);
             switch (tmpProperty) {
                 case SimpleBooleanProperty tmpSimpleBooleanProperty -> {
-                    ComboBox<Boolean> tmpBooleanComboBox = new ComboBox<>();
+                    //implement toggle switch here; write own class that implements a toggle switch
+                    ToggleSwitch tmpToggle = new ToggleSwitch();
+                    tmpToggle.setTooltip(tmpTooltip);
+                    tmpToggle.valueProperty().bindBidirectional(tmpSimpleBooleanProperty);
+                    //add to gridpane
+                    aGridPane.add(tmpToggle, 1, tmpRowIndex++);
+                    GridPane.setMargin(tmpToggle, new Insets(GuiDefinitions.GUI_INSETS_VALUE));
+                    /*ComboBox<Boolean> tmpBooleanComboBox = new ComboBox<>();
                     tmpBooleanComboBox.setPrefWidth(GuiDefinitions.GUI_TEXT_FIELD_PREF_WIDTH_VALUE);
                     tmpBooleanComboBox.setMaxWidth(GuiDefinitions.GUI_SETTINGS_TEXT_FIELD_MAX_WIDTH_VALUE);
                     tmpBooleanComboBox.getItems().addAll(Boolean.FALSE, Boolean.TRUE);
                     tmpBooleanComboBox.valueProperty().bindBidirectional(tmpSimpleBooleanProperty);
                     tmpBooleanComboBox.setTooltip(tmpTooltip);
                     //add to gridpane
                     aGridPane.add(tmpBooleanComboBox, 1, tmpRowIndex++);
-                    GridPane.setMargin(tmpBooleanComboBox, new Insets(GuiDefinitions.GUI_INSETS_VALUE));
+                    GridPane.setMargin(tmpBooleanComboBox, new Insets(GuiDefinitions.GUI_INSETS_VALUE));*/
```

Comment:
The to do comment "implement toggle switch here" and the commented-out code for the old combo box solution can be removed now.